hack/generate-values.sh

```diff
@@ -256,3 +285,15 @@ kpack:
 $( cat ${GCP_SERVICE_ACCOUNT_JSON} | sed -e 's/^/      /' )
 EOF
 fi
+
+if [[ -n "${DOCKERHUB_USERNAME:=}" ]]; then
+  cat <<EOF
+
+kpack:
+  registry:
+    hostname: https://index.docker.io/v1/
+    repository: ${DOCKERHUB_USERNAME}
```

Comment:
Is there ever a case where the operator would want to specify a repository that is not the same as the username?